src/PaymentGateways/PayPalCommerce/AdminSettingFields.php

```diff
@@ -472,7 +473,9 @@ class="button-wrap disconnection-setting<?php echo ! $canShowAccountInformation
                                 <span class="actions">
                                     <button
                                         class="js-give-paypal-disconnect-paypal-account"
-                                        data-mode="<?php echo $paypalSetting->mode; ?>">
+                                        data-mode="<?php echo $paypalSetting->mode; ?>"
+                                        data-nonce="<?php echo wp_create_nonce('give_paypal_commerce_disconnect_account'); ?>"
```

Comment:
Does this need to be escaped using [esc_attr()](https://developer.wordpress.org/reference/functions/esc_attr/)?

Reply:
Yeah I think you're right. Resolved: fd347401